.gitignore

```diff
@@ -12,3 +12,16 @@ src/FieldWarning/Joined_Data/*
 src/FieldWarning/MonoBleedingEdge/*
 src/FieldWarning/UnityCrashHandler64.exe
 src/FieldWarning/UnityPlayer.dll
+src/FieldWarning/Assets/Scene/MicroSplatData/MicroSplatConfig 1.asset
+src/FieldWarning/Assets/Scene/MicroSplatData/MicroSplatConfig 1.asset
+src/FieldWarning/Assets/Scene/MicroSplatData/MicroSplatConfig 1.asset
+src/FieldWarning/Assets/Scene/MicroSplatData/MicroSplatConfig 1_diff_tarray.asset
+src/FieldWarning/Assets/Scene/MicroSplatData/MicroSplatConfig 1_normSAO_tarray.asset
+src/FieldWarning/Assets/Scene/MicroSplatData/MicroSplatConfig 2.asset
+src/FieldWarning/Assets/Scene/MicroSplatData/MicroSplatConfig 2_diff_tarray.asset
+src/FieldWarning/Assets/Scene/MicroSplatData/MicroSplatConfig 2_normSAO_tarray.asset
+src/FieldWarning/Assets/Scene/MicroSplatData/MicroSplatConfig.asset
+src/FieldWarning/Assets/Scene/MicroSplatData/MicroSplatConfig_diff_tarray.asset
+src/FieldWarning/Assets/Scene/MicroSplatData/MicroSplatConfig_normSAO_tarray.asset
+src/FieldWarning/Packages/manifest.json
+src/FieldWarning/ProjectSettings/ProjectVersion.txt
```

Comment:
Did you mean to ignore the last two files as well?

I don't know anything about the microsplat config, if you say it randomly changes and can be regenerated by unity then I'll take you up on it.